pkg/ccr/metaer.go

```diff
@@ -82,13 +82,21 @@ type ReplicaMeta struct {
 	Version    int64
 }
 
+type ColumnDescAll struct {
```

Comment:
```suggestion
type MaterializedIndexDesc struct {
```

The describe tables command returns `MaterializedIndex` in doris.